src/subscription/Captcha.ts

```diff
@@ -91,20 +97,28 @@ function showCaptchaDialog(challenge: Uint8Array, token: string): Promise<string
 		const okAction = () => {
 			let parsedInput = parseCaptchaInput(captchaInput)
 
-			if (parsedInput) {
-				dialog.close()
-
-				locator.serviceExecutor
-					.post(RegistrationCaptchaService, createRegistrationCaptchaServiceData({ token, response: parsedInput }))
-					.then(() => {
-						resolve(token)
-					})
-					.catch((e) => {
-						reject(e)
-					})
-			} else {
+			// User entered an incorrectly formatted time
+			if (parsedInput == null) {
 				Dialog.message("captchaEnter_msg")
+				return
 			}
+
+			// The user entered a correctly formatted time, but not one that our captcha will ever give out (i.e. not *0 or *5)
+			const minuteOnesPlace = parsedInput[parsedInput.length - 1]
+			if (minuteOnesPlace !== "0" && minuteOnesPlace !== "5") {
+				Dialog.message("createAccountInvalidCaptcha_msg")
+				return
+			}
+
+			dialog.close()
+			locator.serviceExecutor
```

Comment:
comment for posterity: (eventually) there should be some class that runs captcha and not just a function and we should inject what we need in it